### ISSUE TITLE
Guidance Process:  Quality & Clarity

### ISSUE THREAD

**Issue by aerospaceindustriesassociation** [WhiteHouse/cyber-acquisitions#37]
Concerns over only inter agency only participation and implication of regulation discovery with respect to Federal Rulemaking Process.  Collaboration (domestic and internationally) is key to cyber disciplines.  The guidance appears to be vague and unclear.  As an example the distinction between government information systems, contractor information systems, and contractor information systems operated on behalf of Government are unstructured and out of context with the paragraphs.  Inconsistency is prevalent throughout the document with questions on the direction or applicability.

**Comment by mnighswander** [comment]
I agree, it's also not very well organized. Recommend re-ordering sections (i.e., move section 3 to section 1 and move section 2 to after Continuous Monitoring.
